search/common.hpp

```diff
@@ -1,7 +1,20 @@
 #pragma once
 
+#include "indexer/categories_holder.hpp"
+
+#include "base/buffer_vector.hpp"
+#include "base/small_set.hpp"
+#include "base/string_utils.hpp"
+
 namespace search
 {
+// The prefix is stored separately.
+// todo(@m, @y) Find a way (similar to TokenSlice maybe?) to unify
+//              the prefix and non-prefix tokens.
+using QueryTokens = buffer_vector<strings::UniString, 32>;
+
+using Locales = base::SafeSmallSet<CategoriesHolder::kMaxSupportedLocaleIndex + 1>;
```

Comment:
Please, convert kMaxSupportedLocaleIndex to uint64_t.

Reply:
not sure why, but done